src/overlays/Modal.tsx

```diff
@@ -14,7 +14,8 @@ export interface ModalProps extends Omit<OverlayProps, "children"> {
   modalClassNames?: string
   headerClassNames?: string
   role?: string
-  scrollable?: boolean
```

Comment:
This will make it a breaking change and when it gets pulled into Bloom at least one Modal will need to be updated. I'm not too concerned about that, but we'll want to make sure that when we pull in the latest ui-components version we don't pull this change in without making the change. Maybe we should create a bloom PR immediately after merging this. 

Also, we will want to make sure Dahlia is aware of this

Reply:
Sounds good! We wouldn't be able to upgrade without Typescript errors so hopefully it won't surprise us.